### PR TITLE
Minor changes on all but `chains` and `transporters`

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### What does this implement/fix? Explain your changes.
+ add docstrings (except for `chains` and `transporters`)
+ Minor other issues

#### Any other comments?
@alirezazolanvari Can you please review commits with [:monocle_face:] at the start of them?
@sepandhaghighi and @AHReccese , commits with [:eye:] at the start are those I need your advice for.

### CODE REVIEW

**Comment by AHReccese** [comment]
@sepandhaghighi King, please review the voice I've sent you in telegram, this PR is now ready for review, I really think it's better to complete doc strings(and the other tasks) in another pr.

**Comment by sepandhaghighi** [comment]
Test discord integration 🚀

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

File: pymilo/pymilo_func.py [hunk 1]
```diff
@@ -7,27 +7,55 @@
...
+    Return sklearn version.
+
+    :return: sklearn version as a str
+    """
     return sklearn.__version__
 
 
 def get_sklearn_data(model):
+    """
+    Return sklearn data by serializing given model
```
Comment (alirezazolanvari) [inline pymilo/pymilo_func.py:20]: A delimiter is needed at the end of the description.

File: pymilo/pymilo_func.py [hunk 2]
```diff
@@ -7,27 +7,55 @@
...
+    :type model: any sklearn's model class
+    :return: sklearn data
+    """
     return transport_linear_model(model, Command.SERIALIZE)
 
 
-def convert_to_sklearn_model(import_obj):
+def to_sklearn_model(import_obj):
+    """
+    Return sklearn model by deserializing imported object
```
Comment (alirezazolanvari) [inline pymilo/pymilo_func.py:31]: A delimiter is needed at the end of the description.

**Review by alirezazolanvari — APPROVED** [review]
@AHReccese Thanks for your effort. Everything looks fine and smooth! LGTM 🚀

**Pull request merged**